src/VisualizationTests/CameraTests.cs

```diff
@@ -48,11 +48,11 @@ public void Camera_GoodSaveData_LoadsCorrectly()
 
             var cam = ((HelixWatch3DViewModel)ViewModel.BackgroundPreviewViewModel).Camera;
 
-            //<Cameras>
+            //<Camera>
```

Comment:
you should not change this - you should add an additional test, you're no longer testing if all old dynamo files will load... You need to maintain that old dynamo xml files with <Cameras> elements will load.